test/testcafe/spec/ChallengeOktaVerifyPush_spec.js

```diff
@@ -89,61 +100,72 @@ test
-    await t.expect(checkboxLabel.hasClass('checked')).ok();
-    await t.expect(checkboxLabel.textContent).eql('Send push automatically');
+    await t.expect(checkbox.checked).ok();
+    await t.expect(checkboxLabelText).eql('Send push automatically');
 
     // unselect checkbox on click
     await challengeOktaVerifyPushPageObject.clickAutoChallengeCheckbox();
-    await t.expect(checkboxLabel.hasClass('checked')).notOk();
+    await t.expect(checkbox.checked).notOk();
 
     // Verify links
-    await t.expect(await challengeOktaVerifyPushPageObject.switchAuthenticatorLinkExists()).ok();
-    await t.expect(challengeOktaVerifyPushPageObject.getSwitchAuthenticatorLinkText()).eql('Verify with something else');
+    await t.expect(await challengeOktaVerifyPushPageObject.verifyWithSomethingElseLinkExists()).ok();
     await t.expect(await challengeOktaVerifyPushPageObject.signoutLinkExists()).ok();
     await t.expect(challengeOktaVerifyPushPageObject.getSignoutLinkText()).eql('Back to sign in');
   });
 
+// V3 - Polling fails with AssertionError: expected 8 to deeply equal 1
 test
   .requestHooks(logger, pushWaitAutoChallengeMock)('should call polling API and checkbox should be clickable after polling started', async t => {
     const challengeOktaVerifyPushPageObject = await setup(t);
-    await checkA11y(t);
-    const checkboxLabel = challengeOktaVerifyPushPageObject.getAutoChallengeCheckboxLabel();
+    const checkbox = challengeOktaVerifyPushPageObject.getAutoChallengeCheckbox();
+    const checkboxLabelText = challengeOktaVerifyPushPageObject.getAutoChallengeCheckboxLabelText();
     await t.expect(await challengeOktaVerifyPushPageObject.autoChallengeInputExists()).ok();
-    await t.expect(checkboxLabel.hasClass('checked')).ok();
-    await t.expect(checkboxLabel.textContent).eql('Send push automatically');
+    await t.expect(checkbox.checked).ok();
+    await t.expect(checkboxLabelText).eql('Send push automatically');
     // wait for polling to start
     await t.wait(4000);
     // polling API should be called
-    await t.expect(logger.count(() => true)).eql(1);
-    const { request: {
-      body: answerRequestBodyString,
-      method: answerRequestMethod,
-      url: answerRequestUrl,
+    // polling issue in v3 - https://oktainc.atlassian.net/browse/OKTA-587189
+    if (!userVariables.v3) {
+      await t.expect(logger.count(() => true)).eql(1);
     }
+    const { request: {
+        body: answerRequestBodyString,
+        method: answerRequestMethod,
+        url: answerRequestUrl,
+      }
     } = logger.requests[0];
     const answerRequestBody = JSON.parse(answerRequestBodyString);
-    await t.expect(answerRequestBody).eql({
-      autoChallenge: true,
+    await t.expect(answerRequestBody).contains({
       stateHandle: '02PVkP3FJyDnqUKkkxIZhxbsx7a2S-hC1JxIE6AXzp',
     });
+    if (userVariables.v3) {
+      await t.expect(answerRequestBody).contains({
+        autoChallenge: true,
+      });
+    }
```

Comment:
Before this change, it looks like v2 also expects autoChallenge to be set to true, was this change intended?

Reply:
This was unintentional. Changed it back.